challenge.html

```diff
@@ -1,75 +1,88 @@
 <!DOCTYPE>
 <html>
   <head>
+
   <meta charset="utf-8">
     <title>Chocolate Pizza</title>
+    <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Yellowtail">
+    <link href='http://fonts.googleapis.com/css?family=Patrick+Hand' rel='stylesheet' type='text/css'>
+    <link href="css/meyer_reset.css" type="text/css" rel="stylesheet" />
+    <link href="css/960_16_col.css" type="text/css" rel="stylesheet" />
     <link href="css/challenge.css" type="text/css"
       rel="stylesheet" />
   </head>
   <body>
-      <header>
-        <h1>
-          <img src="images/logo.png"
+    <div class="container_16">
+      <header class= "clearfix">
+        <img class="grid_1" src="images/logo.png"
                alt="Logo" />
-        </h1>
-        <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Yellowtail">
-        <h1 class="blogname">Delicious</h1>
-        <p class="tagline">THE BEST FOOD BLOG ON THE WEB.</p>
-        <img src="images/fb-icon.png"
-               alt="FB-icon" />
-        <img src="images/twit-icon.png"
-               alt="Twitter-icon" />
-        <img src="images/gp-icon.png"
-               alt="GooglePlus-icon" />
-        <img src="images/insta-icon.png"
-               alt="Instagram-icon" />
-        <img src="images/flic-icon.png"
-               alt="Flickr-icon" />
-        <img src="images/pint-icon.png"
-               alt="Pinterest-icon" />
-        <img src="images/rss-icon.png"
-               alt="RSS-icon" />
+        <section class="blog-head grid_4">
+          <h1>Delicious</h1>
+          <p class="small-all-caps">THE BEST FOOD BLOG ON THE WEB.</p>
+        </section>
+        <section class="social-icons grid_5 push_7">
+          <img src="images/fb-icon.png"
+                 alt="FB-icon" />
+          <img src="images/twit-icon.png"
+                 alt="Twitter-icon" />
+          <img src="images/gp-icon.png"
+                 alt="GooglePlus-icon" />
+          <img src="images/insta-icon.png"
+                 alt="Instagram-icon" />
+          <img src="images/flic-icon.png"
+                 alt="Flickr-icon" />
+          <img src="images/pint-icon.png"
+                 alt="Pinterest-icon" />
+          <img src="images/rss-icon.png"
+                 alt="RSS-icon" />
+      </section>
+      <img class= "greybar container_16" src="images/grey_bar.png" />
```

Comment:
Since you have a container_16 that wraps the entire page content, you don't need to add another.